src/styles/_variables.scss

```diff
@@ -72,6 +73,7 @@ $new-live-color: #e12437; // - refs in related-plugin
   defaultTransitionTime: $default-transition-time;
   bottomBarMaxHeight: $bottom-bar-max-height;
   topBarMaxHeight: $top-bar-max-height;
+  videoFilter: $video-filter;
```

Comment:
maybe be more specific with the naming? like videoFilterClassName or something to makeit more meaningful..

Reply:
i started with that, but then i thought it's weird that it'd be named class when all of the other keys next to it (which describe class names and are created automatically), don't have a "class" postfix

<img width="801" alt="image" src="https://github.com/user-attachments/assets/6f87cf00-f9e7-4152-aca4-6c440447a902">